libraries/AP_Terrain/TerrainGCS.cpp

```diff
@@ -196,6 +196,17 @@ void AP_Terrain::get_statistics(uint16_t &pending, uint16_t &loaded) const
     }
 }
 
+/*
+  get some statistics for TERRAIN_REPORT
+ */
+bool AP_Terrain::get_terrain_spacing(float &terrain_spacing)
+{
+    terrain_spacing = grid_spacing;
+    if (isnan(terrain_spacing)) {
```

Comment:
how do we get NaN from an AP_Int16 grid_spacing?

Reply:
Indeed, there's no way it can be NaN.  I've simplified and renamed the method so that it is an accessor of the grid_spacing parameter.